homeassistant/components/tplink/light.py

```diff
@@ -312,19 +327,29 @@ def _handle_coordinator_update(self) -> None:
 class TPLinkSmartLightStrip(TPLinkSmartBulb):
     """Representation of a TPLink Smart Light Strip."""
 
-    device: SmartLightStrip
+    def __init__(
```

Comment:
We can get rid of this now, but that's something for the `light` platform rewrite :+1: